deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -28,20 +28,28 @@ spec:
         securityContext:
           runAsUser: 1000
         env:
+          # servers
+        - name: KONG_PROXY_LISTEN
+          value: 0.0.0.0:8000, 0.0.0.0:8443 ssl http2
+        - name: KONG_ADMIN_LISTEN
+          value: 127.0.0.1:8444 ssl
+        - name: KONG_STATUS_LISTEN
+          value: 0.0.0.0:8100
+          # DB
         - name: KONG_DATABASE
           value: "off"
+          # runtime tweaks
         - name: KONG_NGINX_WORKER_PROCESSES
           value: "1"
-        - name: KONG_NGINX_HTTP_INCLUDE
-          value: "/kong/servers.conf"
+          # logging
         - name: KONG_ADMIN_ACCESS_LOG
           value: /dev/stdout
         - name: KONG_ADMIN_ERROR_LOG
           value: /dev/stderr
-        - name: KONG_ADMIN_LISTEN
-          value: 127.0.0.1:8444 ssl
-        - name: KONG_PROXY_LISTEN
-          value: 0.0.0.0:8000, 0.0.0.0:8443 ssl http2
+        # - name: KONG_PROXY_ACCESS_LOG
+        # - value: /dev/stdout
```

Comment:
Was commenting these intentional? It looks like this is unrelated.

Reply:
Yes, this is intentional.
Users when they are getting started can use this to log proxy access logs quickly.